devdoc/umocktypes_stdint_requirements.md

```diff
@@ -432,387 +408,3 @@ void umocktypes_free_int64_t(int64_t* value);
 ```
 
 **SRS_UMOCKTYPES_STDINT_01_099: [** umocktypes_free_int64_t shall do nothing. **]**
-
-## umocktypes_stringify_uint_fast8_t
-
```

Comment:
Why remove the fast?

Reply:
Oh I see, it is actually not implemented

---
In reply to: [344956741](https://github.com/Azure/umock-c/pull/134#discussion_r344956741) [](ancestors = 344956741)

Reply:
Yup, cleaned up a lot of the traceabilitytool errors by removing this because none of these seem to exist in code

---
In reply to: [344956920](https://github.com/Azure/umock-c/pull/134#discussion_r344956920) [](ancestors = 344956920,344956741)